pkg/config/load_test.go

```diff
@@ -184,27 +185,58 @@ mirror:
   - catalog: registry.com/ns/baz:v1.2
     targetTag: v1.3
   - catalog: registry.com/ns/baz:v1.2
+  - catalog: oci:///tmp
+  - catalog: oci:///tmp/foo/bar
+  - catalog: oci:///tmp
+    targetName: baz
+  - catalog: oci:///tmp/foo/bar
+    targetName: baz
 `
 
+	// FUTURE: a new PR is likely to change away from TargetName to TargetCatalog
+	// and GetUniqueName is might change too. However, making this into a
+	// table driven test makes sense at this point, so adding that now as well as
+	// additional OCI tests
+	type testCase struct {
+		expected string
+	}
+	tests := []testCase{
+		{
+			expected: "registry.com/ns/bar:v1.2",
+		},
+		{
+			expected: "registry.com/ns/foo:v1.3",
+		},
+		{
+			expected: "registry.com/ns/baz:v1.3",
+		},
+		{
+			expected: "registry.com/ns/baz:v1.2",
+		},
+		{
+			expected: "tmp",
```

Comment:
These tests fail - could you update to expected value i.e for tmp it should be /tmp etc

Reply:
I've updated the tests and added some more scenarios. I've not changed `GetUniqueName` so the results just simply match the value that came out. This is used in a bunch of places, so getting this right is going to be critical. It has to satisfy what was stated in https://issues.redhat.com//browse/CFE-764 and also has to work for renderDC code paths as well as storage code paths.